python/sanlock.c

```diff
@@ -737,9 +790,10 @@ acquired or released.");
 static PyObject *
 py_inq_lockspace(PyObject *self __unused, PyObject *args, PyObject *keywds)
 {
-    int rv, waitrs = 0, flags = 0;
-    const char *lockspace, *path;
+    int rv = -1, waitrs = 0, flags = 0;
+    const char *path;
     struct sanlk_lockspace ls;
+    PyObject *lockspace = NULL;
```

Comment:
I think the order of the local variables should match the usage:

- input arguments (e.g. lockspace)
- internal variables (e.g. struct sanlk_lockspace ls)
- return value (e.g. rv)

The current code does not keep this convention, but at least we can add the new lockpace where it was defined before.

Keeping this order also requires having one declaration per line, which is a good thing, making it harder to create too many variables.

Since this patch is large lets just make sure the new lockspace/resource variables are located near the place they were before the patch.